src/containers/InvitationContainer.tsx

```diff
@@ -13,23 +14,36 @@ interface InvitationContainerProps {
   invitationId: string
 }
 
+const LOADING_TIME = 3000
+
 function InvitationContainer({ invitationId }: InvitationContainerProps) {
+  const [isLoading, setIsLoading] = useState<boolean>(true)
+  const isFetching = useSelector(invitationSelector.getInvitationFetching)
   const dispatch = useDispatch()
   const history = useHistory()
   const invitation = useSelector(invitationSelector.getInvitation)
 
+  const loadInvitatoin = () => {
+    setTimeout(() => {
+      setIsLoading(false)
+    }, LOADING_TIME)
+  }
+
+  const fetchInvitation = useCallback(async () => {
+    try {
+      await dispatch(invitationAction.getInvitation({ invitationId }))
```

Comment:
```suggestion
      await dispatch(invitationAction.getInvitation({ invitationId })).promise
```
제가 지난 리뷰때 놓쳤었나봐요ㅠㅠ 뒤에 promise가 빠져있으면 해당 액션은 프로미스를 호출하는게 아니라 비동기 로직이 정상적으로 작동하지 않습니다 `.promise`만 추가해주면 될것 같습니다

Reply:
반영하였습니다!